DIRECTORY.md

```diff
@@ -153,6 +153,8 @@
     * [Sol2](https://github.com/TheAlgorithms/Ruby/blob/master/project_euler/problem_004/sol2.rb)
   * Problem 005
     * [Sol1](https://github.com/TheAlgorithms/Ruby/blob/master/project_euler/problem_005/sol1.rb)
+  * Problem 006
+    * [Sol1](https://github.com/TheAlgorithms/Ruby/blob/master/project_euler/problem_006/sol1.rb)    
```

Comment:
DIRECTORY.md is not to be updated, as it gets auto-updated automatically (see https://github.com/TheAlgorithms/Ruby/blob/master/CONTRIBUTING.md)

Reply:
```suggestion
```